flow/flow.go

```diff
@@ -135,12 +135,18 @@ func FLowsFromSFlowSample(ft *FlowTable, sample *layers.SFlowFlowSample, probePa
 	for _, rec := range sample.Records {
 
 		/* FIX(safchain): just keeping the raw packet for now */
-		record, ok := rec.(layers.SFlowRawPacketFlowRecord)
-		if !ok {
-			logging.GetLogger().Critical("1st layer is not SFlowRawPacketFlowRecord type")
+		switch rec.(type) {
+		case layers.SFlowRawPacketFlowRecord:
```

Comment:
Well, adding a break will add dead code. It's quite usual to have the default statement at the end, not so readable too (IHMO). If you think moving at the end "case layers.SFlowRawPacketFlowRecord" is mandatory I can do it but I'm not conviced :)